code/game/objects/items/storage/backpack.dm

```diff
@@ -536,11 +536,12 @@
 		new /obj/item/grenade/plastic/x4(src)
 
 /obj/item/storage/backpack/duffelbag/syndie/firestarter
-	desc = "A large duffel bag containing New Russian pyro backpack sprayer, a pistol, a pipebomb, fireproof hardsuit, ammo, and other equipment."
+	desc = "A large duffel bag containing an upgraded flamethrower,spare plasma tank, a pistol, a pipebomb, fireproof hardsuit, ammo, and other equipment."
 
 /obj/item/storage/backpack/duffelbag/syndie/firestarter/PopulateContents()
 	new /obj/item/clothing/under/syndicate/soviet(src)
-	new /obj/item/watertank/op(src)

```

Comment:
Please specify you're removing the spray. This does matter .

Reply:
I did mention this in the commit, but my mistake. I'll endeavor to be more verbose in my pr Description.